boards/arm/adi_eval_adin1110ebz/doc/index.rst

```diff
@@ -0,0 +1,170 @@
+monitoring and diagnostics can be accessed over the USB using serial command interface.
+The board also provides an Arduino interface.
+
+Alternatively, the board can operate in stand-alone mode where it is configured by setting hardware
+configuration links and switches. On-board LEDs provide status indication.
+
+The SPI interface provides configuration and data access to the ADIN1110.
+
+A small prototyping area and test points are provided for experimentation with alternative cable
+connection topologies including isolation transformers and/or power coupling inductors.
+
+
+Hardware
+********
+
+The ADI EVAL-ADIN1110EBZ hardware features list is available here:
+
+https://wiki.analog.com/resources/eval/user-guides/eval-adin1110ebz-user-guide
+
+
+Supported Features
+==================
+
+The ADI adi_eval_adin1110ebz board configuration supports the
+following hardware features:
+
++--------------+------------+-------------------------------------+
+| Interface    | Controller | Driver/Component                    |
++==============+============+=====================================+
+| NVIC         | on-chip    | nested vector interrupt controller  |
++--------------+------------+-------------------------------------+
+| UART         | on-chip    | serial port-polling;                |
+|              |            | serial port-interrupt               |
++--------------+------------+-------------------------------------+
+| PINMUX       | on-chip    | pinmux                              |
++--------------+------------+-------------------------------------+
+| GPIO         | on-chip    | gpio                                |
++--------------+------------+-------------------------------------+
+| I2C          | on-chip    | i2c                                 |
++--------------+------------+-------------------------------------+
+| SPI          | on-chip    | spi                                 |
++--------------+------------+-------------------------------------+
+| PWM          | on-chip    | pwm                                 |
++--------------+------------+-------------------------------------+
+| WATCHDOG     | on-chip    | independent watchdog                |
++--------------+------------+-------------------------------------+
+| ADIN1110     | spi        | adin1110 10BASE-T1L mac/phy         |
++--------------+------------+-------------------------------------+
+| FT232        | uart       | usb-uart                            |
++--------------+------------+-------------------------------------+
+| ADT7422      | i2c        | temperature sensor                  |
++--------------+------------+-------------------------------------+
+| ISS66WVE4M16 | fmc        | 8MB PSRAM                           |
++--------------+------------+-------------------------------------+
+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/adi_eval_adinxxxxebz/adi_eval_adin1110ebz_defconfig``
```

Comment:
Looks like the `xxxx` is left over from when I intended both boards to be in one directory. :wink:

Reply:
thansk, fixed